innerprod/fullysec/damgard_dec_multi_test.go

```diff
@@ -26,18 +26,23 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestFullySec_DamgardDecMulti(t *testing.T) {
+func testFullySec_DamgardDecMultiFromParam(t *testing.T, param damgardTestParam) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [don't use underscores in Go names; func testFullySec_DamgardDecMultiFromParam should be testFullySecDamgardDecMultiFromParam](https://app.codacy.com/app/tilenmarc/gofe_2/pullRequest?prid=4432579)